### PR TITLE
Hw02 task03

### PR DESCRIPTION
**Pull request by julwein**

### CODE REVIEW

File: homework_02/task_03.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    [1, 4],
+    [2, 3],
+    [2, 4],
+]
+"""
+from itertools import product
+from typing import Any, List
+
+
+def combinations(*args: List[Any]) -> List[List]:
```
Comment (Cjkjvfnby) [inline homework_02/task_03.py:17]: Please add tests.

File: homework_02/task_03.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    """
+    Return a list of all possible lists made up of elements from the input lists.
+
+    Test from the example
+    >>> combinations([1, 2], [3, 4])
+    [[1, 3], [1, 4], [2, 3], [2, 4]]
+    """
+    outp_lst = []
+    for tup in product(*args):
```
Comment (Cjkjvfnby) [inline homework_02/task_03.py]: Let's do it in a low-level way, without libs.

**Review by Cjkjvfnby — CHANGES_REQUESTED** [review]

**Review by Cjkjvfnby — APPROVED** [review]
Ok.

Bonus task: solve it with the `itertools` module.

**Comment by julwein** [comment]
> Bonus task: solve it with the `itertools` module.

Done - that was the first solution. Well, nearly, a month ago...

**Pull request merged**